### PR TITLE
Boss

### PR DESCRIPTION
**Pull request by hotblack86**

### CODE REVIEW

File: src/constants.py [hunk 1]
```diff
@@ -20,14 +20,17 @@
...
+
 
 TILESIZE = 60
 GRIDWIDTH = WIDTH / TILESIZE
 GRIDHEIGHT = HEIGHT / TILESIZE
 
 HERO_HEALTH = 100
-MOB_DAMAGE = 34
+MOB_DAMAGE = 20
+BOSS_DAMAGE = 50
```
Comment (CKKH) [inline src/constants.py]: :cheese:

File: src/boss.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        if not collide(self, self.game.walls_sprites, d_x, d_y) and not collide(
+                self, self.game.all_sprites, d_x, d_y, self.boss_touches_hero):
+            self.x_pos += d_x
+            self.y_pos += d_y
+
+    def boss_touches_hero(self):
+        ''' Updates Hero Health '''
+
+        print("Boss hit hero")
+        self.game.hero.health -= BOSS_DAMAGE
```
Comment (michaelnguyen974) [inline src/boss.py:49]: Nice

**Review by michaelnguyen974 — APPROVED** [review]

**Review by matharotheelf — APPROVED** [review]
Boss man

**Review by Sindex42 — APPROVED** [review]
Looks good, would like to refactor if there is time

**Pull request merged**